templates/snap-details.html

```diff
@@ -29,14 +29,28 @@
           <div class="p-media-object__details">
             <h1>{{ snap_title }}</h1>
             <p class="p-media-object__content--large">{{ publisher }}</p>
-
-            <p class="p-media-object__content--large">
-              {% if is_linux %}
-                <a class="p-button--positive" href="snap://{{ package_name }}" onclick="if(typeof ga !== 'undefined'){ga('gtm1.send', {hitType: 'event', eventCategory: 'Snap details', eventAction: 'Clicked Install button', eventLabel: 'Install {{ package_name }}'})}">Install</a>
-              {% else %}
-                You can install this software on a snap-powered system like <a class="p-link--external" href="https://www.ubuntu.com/download">Ubuntu</a>.
-              {% endif %}
-            </p>
+            <div class="row p-media-object__content--large snapcraft-snap-details__snippet">
+              <div class="col-6">
+                <div class="p-code-snippet">
+                  <input
+                    class="p-code-snippet__input"
+                    id="snap-install"
+                    value="sudo snap install {{ package_name }}"
+                    readonly="readonly"
+                    oncopy="if(typeof ga !== 'undefined'){ga('gtm1.send', {hitType: 'event', eventCategory: 'Snap details', eventAction: 'Copy CLI install instructions', eventLabel: 'Copy {{ package_name }}'})}">
```

Comment:
Does `oncopy` this work (crossbrowser)?
Does copying through `clipboard` script trigger it?

Reply:
I've changed this to use the built-in event for the copy button and added some wonderful event js for the input instead. I don't think there's any way around having two events @caldav...
MDN says oncopy is 'non-standard' https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/oncopy so we should avoid using it.

Reply:
Ok, let's see how it goes. The likeliness of triggering both is lower than before, thanks to this creative function :)